### ISSUE TITLE
History stops adding to list of transactions

### ISSUE THREAD

**Issue by jsonnentag** [monerov/monerov-gui#8]
The last 2 times I've checked (last week and just now) I had to restart the GUI to get the latest transactions (payments received) to appear on the History listing.  The Balance is updated OK, but the History doesn't list the latest transactions (mining payments).  After quitting and then starting the GUI they do appear on the list.

**Comment by vanfactory** [comment]
That's probably because your monerov daemon wasn't synched to the latest height. Unrelated to the GUI.

**Comment by jsonnentag** [comment]
That doesn't make the least bit of sense!  It is already stated that the TOTAL available balance is accurate and listed as available to spend, but the latest transactions that get it up to that total were not all listed until the GUI was restarted. . . .  Try again. . . .

**Comment by vanfactory** [comment]
When you restart the GUI it also restarts the daemon that is why.

**Comment by jsonnentag** [comment]
No.  As listed in another problem (#6), the GUI does not start the daemon anymore.  The daemon EXE file has to be started separately.  In any case, when the GUI was shut down and asked if the daemon was to be shut down it (the daemon) was not shut down.  The window it is running in remained open and active.  The block number listed at the bottom left corner of the GUI was already up to the then current level as well.

When the current wallet balance is accurate and the block level is accurate (most recent listed as sync'ed) and the only thing not listed right is the History of transactions the daemon isn't the problem. . . .

**Comment by jsonnentag** [comment]
Here, you can see a sequence from the last payment received.
 - [12:50 PM] First is a view of the daemon running. . . .
 - [12:50 PM] Next a view of the History and total wallet value
 - GUI was then stopped and [12:54 PM] restarted (daemon NOT stopped during GUI shutdown), and you can now see the latest payment that got the total balance up to the value that was seen before but NOT listed in the history
 - [12:57 PM] Finally, another view of the daemon still running as before

![daemon_12_50_pm](https://user-images.githubusercontent.com/36777806/41681581-e554dc56-7489-11e8-9b51-76736cf2edcb.jpg)
![gui_history_12_50_pm](https://user-images.githubusercontent.com/36777806/41681599-ef1caf2a-7489-11e8-801a-b4cfea9efb74.jpg)
![gui_history_12_54_pm](https://user-images.githubusercontent.com/36777806/41681605-f3005de4-7489-11e8-84a9-e8e75ddfad97.jpg)
![daemon_12_57_pm](https://user-images.githubusercontent.com/36777806/41681613-f7cd609c-7489-11e8-9d6f-e49d8ae5a72f.jpg)
